src/model/StringCalculator.js

```diff
@@ -0,0 +1,24 @@
+export class StringCalculator {
+  separateCustomDelimiter(input) {
+    const customDelimiterRegex = /^\/\/(.)\\n/;
+    const customemDelimiterFormat = input.match(customDelimiterRegex);
+    const customDelimiter = customemDelimiterFormat[1];
+    const valueString = input.slice(customemDelimiterFormat[0].length);
+    return { customDelimiter, valueString };
+  }
+
+  extractValues(delimiters, valueString) {
+    const delimiterRegex = new RegExp(`[${delimiters.join('')}]`);
+    const values = valueString.split(delimiterRegex);
+    return values;
+  }
+
+  calculateSum(values) {
+    let sumValue = 0;
+    if (values === '') return sumValue;
+    for (let value of values) {
+      sumValue = sumValue + Number(value);
+    }
+    return sumValue;
```

Comment:
```suggestion
return values.reduce((acc, value) => acc + Number(value), 0);
```
js에서는 합을 구할 때 reduce를 사용합니다. 일반적인 for 반복문보다 코드를 더 짧고 명확하게 작성할 수 있습니다. 

JS의 고차 함수 forEach, reduce 등을 적극적으로 활용하면 좋습니다ㅎㅎ!